tests/testthat/test-object_usage_linter.R

```diff
@@ -765,3 +765,42 @@ test_that("functional lambda definitions are also caught", {
     object_usage_linter()
   )
 })
+
+test_that("messages without location info are repaired", {
```

Comment:
one more test to add if you don't mind:

when the object usage comes from the formals

```r
foo <- function(a = bar()) {
  a
}
```